File: Selenium/Activities/Activity11.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver import Keys, ActionChains

with webdriver.Firefox() as driver:
    driver.get("https://training-support.net/webelements/keyboard-events")
    print(f"The page title is : {driver.title}")
    builder = webdriver.ActionChains(driver)
    builder.send_keys("Typing from Selenium").send_keys(Keys.RETURN).perform()
    txt = driver.find_element(By.XPATH,"//p[@id='result']/following-sibling::h1")
    print(f"The confirmation text is : {txt.text}")
    driver.quit()

File: Selenium/Activities/Activity11.java
package examples;

import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.interactions.Actions;
import org.openqa.selenium.By;
import org.openqa.selenium.Keys;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;

public class Activity11 {

	public static void main(String[] args) {

			WebDriver driver = new FirefoxDriver();
			
			driver.get("https://training-support.net/webelements/keyboard-events");
			System.out.println("The page title is : "+ driver.getTitle());
			
			
			Actions builder = new Actions(driver);
			
			builder.sendKeys("Typing from Selenium").sendKeys(Keys.RETURN).build().perform();
			WebElement txt = driver.findElement(By.xpath("//p[@id='result']/following-sibling::h1"));
			System.out.println("The confirmation text is : "+txt.getText());
			
			driver.quit();

	}

}
